### PR TITLE
Customer object

### PR DESCRIPTION
**Pull request by candrasaputra**
- enable SMS / WA notification

### CODE REVIEW

File: Xendit/M2Invoice/Controller/Checkout/Invoice.php [hunk 1]
```diff
@@ -74,7 +76,12 @@ private function getApiRequestData($order)
             'payment_methods'       => json_encode([strtoupper($preferredMethod)]),
             'platform_callback_url' => $this->getXenditCallbackUrl(),
             'success_redirect_url'  => $this->getDataHelper()->getSuccessUrl(),
-            'failure_redirect_url'  => $this->getDataHelper()->getFailureUrl($orderId)
+            'failure_redirect_url'  => $this->getDataHelper()->getFailureUrl($orderId),
+            'customer'              => (object) [
+                'given_names'   => $order->getCustomerFirstname(),
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/Controller/Checkout/Invoice.php:81]: if have last name can also append it here

File: Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php [hunk 2]
```diff
@@ -78,7 +80,12 @@ public function execute()
                 'payment_methods'       => json_encode([strtoupper($preferredMethod)]),
                 'platform_callback_url' => $this->getXenditCallbackUrl(),
                 'success_redirect_url'  => $this->getDataHelper()->getSuccessUrl(true),
-                'failure_redirect_url'  => $this->getDataHelper()->getFailureUrl($orderIncrementIds, true)
+                'failure_redirect_url'  => $this->getDataHelper()->getFailureUrl($orderIncrementIds, true),
+                'customer'              => (object) [
+                    'given_names'   => $billingFirstName,
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php:85]: here as well
PS: No `addresses` input in customer payload?

**Comment by rendegosling** [comment]
looks generally ok, whats the context for this changes?

**Comment by candrasaputra** [comment]
@rendegosling this will enable SMS / WA notification

File: Xendit/M2Invoice/Controller/Checkout/Invoice.php (hunk 1 above)
Comment (candrasaputra) [reply Xendit/M2Invoice/Controller/Checkout/Invoice.php:81]: it's not required and there no country code so i just afraid it's will throw error.

File: Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php (hunk 2 above)
Comment (candrasaputra) [reply Xendit/M2Invoice/Controller/Checkout/InvoiceMultishipping.php:85]: it's not required and there no country code so i just afraid it's will throw error.

**Review by rendegosling — APPROVED** [review]

**Review by kevinalfianto — APPROVED** [review]

**Review by IreneGohtami — APPROVED** [review]

**Pull request merged**